test.js

```diff
@@ -0,0 +1,94 @@
+const fs = require('fs-extra');
```

Comment:
Did you accidentally commit this file? It's just the same as cleanupTranslations.js

Reply:
yeah Sorry wait i am fixing